docs/code-of-conduct.rst

```diff
@@ -0,0 +1,33 @@
+Code of Conduct
+===============
+
+Like the technical community as a whole, the Write the Docs team and community is made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching and connecting people.
```

Comment:
I'm not sure about this para -- only the open source tech community includes professionals and volunteers, and "mission" implies that we've defined one... maybe borrow from the main writethedocs.org page:

"Write the Docs is a global community of documentarians who share information, discuss ideas, and work together to improve the art and science of documentation.